webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/render/model/EntityAttributes.java

```diff
@@ -30,10 +30,12 @@
     public static final String ATTR_LABEL = "l";
     public static final String ATTR_COLOR = "c";
     public static final String ATTR_HOVER_TEXT = "h";
+    public static final String ATTR_ACTION_BUTTONS = "a";
     
     private @JsonProperty(ATTR_LABEL) String labelText;
     private @JsonProperty(ATTR_COLOR) String color;
     private @JsonProperty(ATTR_HOVER_TEXT) String hoverText;
+    private @JsonProperty(ATTR_ACTION_BUTTONS) Boolean actionButtons;
```

Comment:
Can we use something like this? https://stackoverflow.com/a/37550480/2511197?

Any maybe something like https://stackoverflow.com/a/14447933/2511197 to avoid serializing `0/false` as the default value? Then it wouldn't be necessary to use `Boolean` here.